.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```

Comment:
It doesn't build the source code but build contracts. I think this is a copy/paste error from anywhere CI and you can remove it.